src/cameras/src/pm/Camera.cpp

```diff
@@ -335,12 +335,6 @@ bool pm::Camera<F>::Close() {
         spdlog::error("Failure releasing frame info structure for camera {}", ctx->info.name);
     }
 
-    //TODO Delete buffer
-    if (ctx->curFrameInfo) {
```

Comment:
why is this removed?

Reply:
That seemed to be the line causing the heap error when the app shuts down. I also so this in the lines above that seemed to be doing the same thing
```
if (PV_OK != pl_release_frame_info_struct(ctx->curFrameInfo)) {
    spdlog::error("Failure releasing frame info structure for camera {}", ctx->info.name);
}
```